### PR TITLE
feat: allow wildcard notZeroBaseds value

### PR DESCRIPTION
**Pull request by qkdreyer**

### CODE REVIEW

File: src/dn/heaps/assets/Atlas.hx [hunk 1]
```diff
@@ -135,7 +135,7 @@ class Atlas {
 				if( frameReg.match(k) ) {
 					k = frameReg.matched(1);
 					f = Std.parseInt(frameReg.matched(3));
-					if( notZeroBaseds.exists(k) )
+					if( notZeroBaseds.exists('*') || notZeroBaseds.exists(k) )
```
Comment (kLabz) [inline src/dn/heaps/assets/Atlas.hx:138]: Shouldn't `notZeroBaseds.exists('*')` be tested outside the loop?

**Comment by deepnight** [comment]
Hi! What does this change actually do? 🤔

**Comment by qkdreyer** [comment]
Hey,

That was introduced to be able to load an full zero based Atlas project file by calling :

`dn.heaps.assets.Atlas.load(hxd.Res.file.atlas, () -> {}, ['*'])`

Without knowing all the group names inside the Atlas file

**Comment by deepnight** [comment]
I see, thank you!

**Pull request merged**